src/codeenginestudio/huong/Exercise1.java

```diff
@@ -0,0 +1,47 @@
+package codeenginestudio.huong;
+
+import java.util.Arrays;
+
+public class Exercise1 {
+    public static void main(String[] args) {
+
+        String[] strArray = {"is2", "pa6" ,"Thi1s", "T4est", "3a", "th5"};
+        sortString(strArray);
+    }
+
+    public static boolean isNumeric(String str) {
+        try {
+            Integer.parseInt(str);
+            return true;
+        } catch(NumberFormatException e){
+            return false;
+        }
+    }
+
+    public static char[] convertToArray (String str){
+        char[] charArray = str.toCharArray();
+        return charArray;
```

Comment:
In this function, the variable charArray used to return. Pls, use return str.toCharArray() for instead!